dao/searchDao.js

```diff
@@ -7,7 +7,45 @@ exports.getAll = function(filter, cb) {
     if(filter.class) {
         classFilter = ' AND class = ' + db.escape(filter.class);
     }
-    let sqlQuery = 'SELECT * FROM tbl_tickets WHERE reserver IS NULL' + classFilter + ';';
+
+    // filter to be used on sql query
+    let seatFilter = '';
+    let seatArray = null;
+    // still in string form after getting passed as a query parameter
+    // if there is something the seat object
+    if(filter.seat) {seatArray = filter.seat.split(',');}
+
+    if(Array.isArray(seatArray)) {
```

Comment:
Why not combine this `if` block with the previous? `seatArray` is only an array if `filter.seat` is present ...

Reply:
Yes, you are right. I do not need that.

Reply:
I will implement this in #7.